res/xml/armenian_ph_am.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<keyboard name="Armenian" script="armenian">
+    <row>
```

Comment:
The layout includes all ASCII punctuations except `\\` and `` ` ``. Is it intended ?

Reply:
\\ is included.
` will be added